distribution/alliance/src/main/filtered-resources/etc/org.codice.ddf.admin.application.service.impl.ApplicationServiceImpl.config

```diff
@@ -1,14 +1,14 @@
-ignoredApplications=[
-    "hibernate-osgi",
-    "hibernate-validator-osgi-features",
-    "install-profiles",
-    "kernel",
-    "opendj-embedded",
-    "org.ops4j.pax.cdi",
-    "org.ops4j.pax.jdbc",
-    "org.ops4j.pax.web",
-    "spring",
-    "standard",
-    "alliance-branding",
-    "karaf-decanter"
-]
\ No newline at end of file
+ignoredApplications = [ \
+  "hibernate-osgi", \
+  "hibernate-validator-osgi-features", \
+  "install-profiles", \
+  "kernel", \
+  "opendj-embedded", \
+  "org.ops4j.pax.cdi", \
+  "org.ops4j.pax.jdbc", \
+  "org.ops4j.pax.web", \
+  "spring", \
+  "standard", \
+  "karaf-decanter", \
+  "alliance-branding", \
```

Comment:
✏️  Remove trailing `,`.

Reply:
I decided to leave the extra comma and include a note in the Documentation. See https://github.com/codice/ddf/pull/2241/commits/5e3264137917b6ce6ffed300a59c64405933b8e4#r135342949.